front_end/panels/ai_chat/core/EnvironmentConfig.ts

```
@@ -0,0 +1,336 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * Environment Configuration Manager for AI Chat Panel
+ * 
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)